martin/src/pg/config_table.rs

```diff
@@ -25,6 +25,12 @@ pub struct TableInfo {
     /// Geometry SRID
     pub srid: i32,
 
+    /// Projection of SRID if found in `spatial_ref_sys`
+    pub proj: Option<String>,
+
+    /// Projection unit of SRID if found in `spatial_ref_sys`
+    pub proj_unit: Option<String>,
```

Comment:
I am not quite sure about the naming. I usually like to not hamming-code these things. Maybe `srid_projection`, `srid_projection_unit`?
But I think @nyurik has a better taste in this.

An alternative configuration format if these are srid-specific would be to set this as:
`srid: {}`